### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by basil**
As of the latest plugin parent POM, the properties are already defined in the plugin parent POM and the BOMs are already imported, so these code is duplicate and can be safely deleted.

### CODE REVIEW

**Comment by basil** [comment]
Build is green: https://ci.jenkins.io/job/Plugins/job/analysis-pom-plugin/job/PR-781/4/

File: pom.xml [hunk 1]
```diff
@@ -193,7 +167,9 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
+        <version>${maven-enforcer-plugin.version}</version>
```
Comment (uhafner) [inline pom.xml:172]: Isn't this version already defined in the parent pom, then we do not need it here?

File: pom.xml [hunk 2]
```diff
@@ -216,7 +192,9 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
```
Comment (uhafner) [inline pom.xml:197]: same here

File: pom.xml [hunk 3]
```diff
@@ -226,6 +204,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>${maven-jar-plugin.version}</version>
```
Comment (uhafner) [inline pom.xml:207]: same here

File: pom.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.69</version>
+    <version>4.73</version>
```
Comment (uhafner) [inline pom.xml:7]: Unrelated change already part of dependabot.

**Review by uhafner — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (basil) [reply pom.xml:7]: But you haven't merged it, yet? So as long as trunk doesn't contain it, it is a valid change.

File: pom.xml (hunk 1 above)
Comment (basil) [reply pom.xml:172]: I am just being consistent with your other code in this POM file that already contains redundant versions. If you think the redundancy should be eliminated, remove it from your existing code in a separate PR and then I will follow suit in this PR.

File: pom.xml (hunk 2 above)
Comment (basil) [reply pom.xml:197]: I am just being consistent with your other code in this POM file that already contains redundant versions. If you think the redundancy should be eliminated, remove it from your existing code in a separate PR and then I will follow suit in this PR.

File: pom.xml (hunk 3 above)
Comment (basil) [reply pom.xml:207]: I am just being consistent with your other code in this POM file that already contains redundant versions. If you think the redundancy should be eliminated, remove it from your existing code in a separate PR and then I will follow suit in this PR.

**Comment by basil** [comment]
I am denying the request for changes, for the reasons explained above.

**Review by uhafner — APPROVED** [review]
